src/app/views/components/select/select.module.ts

```diff
@@ -1,24 +1,22 @@
-import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
-import { defineSelectComponent } from '@tylertech/forge';
-import { ForgeSelectModule } from '@tylertech/forge-angular';
+import { ForgeOptionModule, ForgeSelectModule, ForgeSelectProxyModule } from '@tylertech/forge-angular';
 
 import { SharedModule } from '../../../shared/shared.module';
 import { SelectRoutingModule } from './select-routing.module';
 import { SelectComponent } from './select.component';
 
-defineSelectComponent();
-
 @NgModule({
   declarations: [SelectComponent],
   imports: [
     CommonModule,
     SelectRoutingModule,
     FormsModule,
     SharedModule,
-    ForgeSelectModule
-  ],
-  schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    ForgeOptionModule,
+    ForgeSelectModule,
```

Comment:
It may also make sense to export ForgeOptionModule from the ForgeSelectModule, and similarly with tab/tab-bar and view/view-switcher, etc., bundling up the components that are commonly used in conjunction with each other for convenience.